ddtrace/bootstrap/sitecustomize.py

```diff
@@ -119,6 +120,11 @@ def update_patched_modules():
         env_tags = os.getenv("DD_TRACE_GLOBAL_TAGS")
         tracer.set_tags(parse_tags_str(env_tags))
 
+    # instrumentation telemetry writer should be enabled/started after the global tracer and configs
+    # are initialized
+    if asbool(get_env("instrumentation_telemetry", "enabled")):
```

Comment:
The only way to enable telemetry is by using `ddtrace-run`? is that what we want?

Reply:
Yes the only way to enable telemetry automatically is to use `ddtrace-run`. A user can enable telemetry manually using `from ddtrace.telemetry import InstrumentationTelemetry; InstrumentationTelemetry.enable()`. We also use that pattern in `RuntimeMetrics`

Reply:
This is fine for now, we can address again before we enable by default.